modules/fedimint-lnv2-server/src/lib.rs

```diff
@@ -544,6 +557,37 @@ impl ServerModule for Lightning {
                     Ok(module.outgoing_contract_expiration(db, contract_id).await)
                 }
             },
+            api_endpoint! {
+                ADD_GATEWAY_ENDPOINT,
+                ApiVersion::new(0, 0),
+                async |_module: &Lightning, context, gateway: SafeUrl| -> bool {
+                    check_auth(context)?;
+
+                    let db = context.db();
+
+                    Ok(Lightning::add_gateway(db, gateway).await)
+                }
+            },
+            api_endpoint! {
+                REMOVE_GATEWAY_ENDPOINT_V2,
+                ApiVersion::new(0, 0),
```

Comment:
Should we increment minor api version of this module so clients can detect new calls, and then `ApiVersion` here needs to be set to the new version.

Reply:
I don't follow, do you want to change something here?

Reply:
@dpc there are no stability guarantees around LNv2 for now, so I don't think that's necessary.

Reply:
Oh, right.